django_refugeerights/subscription/tests.py

```diff
@@ -85,6 +85,22 @@ def test_create_subscription(self):
         self.assertEqual(d.lang, "en_GB")
         self.assertEqual(d.schedule.id, self.schedule.id)
 
+    def test_switch_subscription(self):
+        post_data = {
+            'contact_key': 'ghfdhftesdf',
+            'messageset_id': '1',
+            'schedule': self.schedule.id
+        }
+        response = self.client.post('/subscription/switch_subscription/',
+                                    json.dumps(post_data),
+                                    content_type='application/json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        d = Subscription.objects.last()
+        self.assertEqual(d.contact_key, "ghfdhftesdf")
+        self.assertEqual(d.messageset_id, 1)
+        self.assertEqual(d.schedule.id, self.schedule.id)
```

Comment:
For this test, before the `post` request, we probably want to check that the existing (old) subscription is there and active, and after the `post`, we want to check that the old subscription is there and inactive.